mysql-test/suite/tokudb_rpl/r/rpl_tokudb_stm_mixed_crash_safe.result

```diff
@@ -203,6 +203,8 @@ END|
 
 
 STOP SLAVE SQL_THREAD;
+Warnings:
```

Comment:
Here and everywhere in this file
Probably it's better to change the command in the .test file

Reply:
Explained above